jenkins/common.groovy

```diff
@@ -266,47 +266,6 @@ def build(String ref, dockerContainer, String assetSource, String envName, Boole
   }
 }
 
-def buildAll(String ref, dockerContainer, Boolean contentOnlyBuild) {
```

Comment:
>Since we're no longer using buildAll, can we take it out of /jenkins/common.groovy? - https://github.com/department-of-veterans-affairs/vets-website/pull/16965#discussion_r622179954

Reply:
Removing buildAll makes sense to me since it eliminates confusion as to what code is actually being used